modules/test/playwright/pages/dynamic-data-mapping-form-web/FormsPage.ts

```diff
@@ -67,7 +67,7 @@ export class FormsPage {
 		this.managementToolbarSelectAllItems = page.getByLabel(
 			'Select All Items on the Page'
 		);
-		this.newFormButton = page.getByRole('link', {name: 'New Form'});
+		this.newFormButton = page.getByText('New', {exact: true});
```

Comment:
The old case `'New Form'` is still working? We are removing the term `'Form'` and using the `exact:true`. Could you confirm that everything is ok with the test:

**_Select from list with multiple selections allowed is auto-filled by data provider defined in rule_**

Reply:
Hey, thanks, making this change was a mistake. I did not want to rely on the empty state "New Form" button because if a cleanup of a previous test wasn't completed correctly, this button would not be available. But there is already a [method to click on the management toolbar new button in this same class](https://github.com/liferay/liferay-portal/blob/master/modules/test/playwright/pages/dynamic-data-mapping-form-web/FormsPage.ts#L99). I have already update this PR.